### DISCUSSION TITLE
Feature request? Show imports

### DISCUSSION THREAD

**Discussion by thany** [webpack-contrib/webpack-bundle-analyzer#569] [Ideas]
#### Issue description
This analyzer seems to be limited to whole modules. I would like to see which functions/etc are imported from those modules.

I'm not sure if this is a feature request or a bug report. If WBA has this functionality and it's not working for me, or if WBA doesn't have it, I don't really know. I suspect the latter.

#### Technical info
* Webpack Bundle Analyzer version: 3.3.2
* Webpack version: 4.3
* Node.js version: 10.13
* npm/yarn version: 6.4.1/1.16
* OS: Windows 10

#### Debug info
To keep it simple, I've used the CLI tool:
```
webpack ./src/js/main.ts --profile --json > stats.json
webpack-bundle-analyzer stats.json -m static
```
I've also implemented this in webpack via gulp, which produces the same output. But it's probably easier to stick with CLI so we can rule out other components.

[My stats.json (zipped)](https://github.com/webpack-contrib/webpack-bundle-analyzer/files/3290036/stats.zip)

**Comment by valscion** [comment]
Seems similar to #53, #41 and #269

**Comment by thany** [comment]
Skimming through those feature requests, it seems they are mostly about what includes something (or why is something included), not so much what parts from a given import are actually used.
But I could be wrong of course.

**Comment by th0r** [comment]
> not so much what parts from a given import are actually used

Could you explain what exactly do you want to see and how it should be shown?
What is your usecase? Maybe some examples?
